src/android/SplashScreen.java

```diff
@@ -341,7 +341,11 @@ public void onCancel(DialogInterface dialog) {
                     }
                 });
 
-                spinnerDialog.setCancelable(false);
+                if(preferences.getBoolean("spinnerCancelable",true)){
```

Comment:
Why a user would need to cancel the spinner? This will not dismiss the splashscreen itself.
Could you please explain what is your scenario?